src/Authorization.AspNetCore/IAuthorizationErrorMessageBuilder.cs

```diff
@@ -0,0 +1,32 @@
+#nullable enable
+
+using System.Text;
+using GraphQL.Language.AST;
+using Microsoft.AspNetCore.Authorization;
+
+namespace GraphQL.Server.Authorization.AspNetCore;
+
+public interface IAuthorizationErrorMessageBuilder
+{
+    /// <summary>
+    /// Generates an authorization error message based on the provided <see cref="AuthorizationResult"/>
+    /// </summary>
+    /// <param name="operationType">The GraphQL operation type.</param>
+    /// <param name="result">The <see cref="AuthorizationResult"/> which is used to generate the message.</param>
+    /// <returns></returns>
```

Comment:
```suggestion
```

Reply:
ahhhh if you do it fast, you do it twice... sorry